component-sets/default-components/components-definition.json

```diff
@@ -279,6 +280,19 @@
             "dataType": "inlineStyles",
             "featureFlag": "ContentStation-LocalStyleOverrides"
         },
+        {
+            "name": "chart-height",
+            "label": { "key": "CHART_HEIGHT__LABEL" },
+            "control": {
+                "type": "text",
+                "pattern": "^([0-9]{1,3}([.][0-9]{1,})?|auto)$",
+                "defaultValue": "",
+                "unit": "em",
+                "inputPlaceholder": { "key": "PROPERTY_AUTO_NUMBER_PLACEHOLDER" }
+            },
+            "selector": "[doc-chart]",
+            "dataType": "inlineStyles"
```

Comment:
Is the value indeed applied as inline style? See [here](https://github.com/WoodWing/csde-components-boilerplate/blob/604b2a0823035e11fa987cbcac4ceec58d063636/docs/PROPERTIES.md#data-types)

Reply:
nope, changed it to `data`